05_syntax_lab/01.py

```diff
@@ -1,5 +1,11 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+"""
+Exercise 1
+
+- Largest number between 10 numbers
+"""
+
+
+num = []
+for i in range(1,11):
+	num.append(raw_input('Please enter the %s number: ' % i))
+print 'The Largest number entered is: ', max(num)
```

Comment:
Hi Rony and welcome to the exercises!

Nice solution with some advanced tools.
👍